### ISSUE TITLE
Page region missing an accessible name in CTA Group component

### ISSUE THREAD

**Issue by poojagunturu96** [middlebury/midd-frontend#656]
Any page section with the role of region should also have an accessible name to identify it.

![Screenshot 2024-09-24 at 6 51 25 PM](https://github.com/user-attachments/assets/ccf5b349-2f8d-450e-be79-88a4be15ee73)

Siteimprove report: https://my2.siteimprove.com/Accessibility/7333/NextGen/Issue/1?conformance=&siteTargetIssueKinds=1,2&wcagVersion=21&pageSegments=&ruleName=sia-r40&ruleId=40&issueKind=1&exceptTags=1,2&siteTarget.wcagVersion=21&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2
